flashinfer/moe_ep/kernel_src/sm90/push_style_megamoe/shim/nvfp4_runner.py

```
@@ -0,0 +1,1201 @@
+        self.runner.run(
+            activation[: self.max_rows, : self.k].view(torch.float8_e4m3fn),
+            scales,
+            offsets,
+            out=output[: self.max_rows, : self.n],
+            trusted_offsets=True,
+            prepare_schedule=prepare_schedule,
+        )
+
+
+class _W4A8PairEngine(Protocol):
+    I: int
```

Ruff (0.16.1)

[error] Ambiguous variable name: `I`

(E741)